### ISSUE TITLE
ImportError: sys.meta_path is None, Python is likely shutting down

### ISSUE THREAD

**Issue by Ze-Yang** [yanxp/MetaR-CNN#12]
![image](https://user-images.githubusercontent.com/42920710/67629989-bb361500-f8ba-11e9-9e89-088c1b8d1fa8.png)
When I run this code, I encounter this error. I don't know how to fix it. Your help is highly appreciated.

**Comment by Hxx2048** [comment]
i  meet this error, too.
Have you solved the problem?

**Comment by EasitMickly** [comment]
> That's maybe your environment is not right.

I have meet the same problem, Can you share the following software versions: Cython easydict matplotlib opencv-python pyyaml scipy tensorboardX, thank you.

**Comment by EasitMickly** [comment]
> That's maybe your environment is not right.

I have met the same problem, Can you share the following software versions: Cython easydict matplotlib opencv-python pyyaml scipy tensorboardX, thank you.

**Comment by cxcjcj** [comment]
> i meet this error, too.
> Have you solved the problem?
Hi,@Hxx20048

I encountered the same problem when running my own dataset. However, I did not encounter this problem when running the author dataset. Have you solved the problem? Could you give me some suggestions?
 
 Thank you very much.

**Comment by cxcjcj** [comment]
I encountered the same problem when running my own dataset. However, I did not encounter this problem when running the author dataset. Have you solved the problem? Could you give me some suggestions?

Thank you very much.

**Comment by Ze-Yang** [comment]
Yes. This is actually due to lib import problem. You should not add lib. in front of the import directory, which makes it malfunction.

**Comment by cxcjcj** [comment]
Thanks for your reply. Can you tell me which files need to be modified?------------------ 原始邮件 ------------------
发件人: "Ze-Yang"<notifications@github.com>
发送时间: 2019年12月7日(星期六) 晚上9:18
收件人: "yanxp/MetaR-CNN"<MetaR-CNN@noreply.github.com>;
抄送: "cxcjcj"<876031584@qq.com>;"Comment"<comment@noreply.github.com>;
主题: Re: [yanxp/MetaR-CNN] ImportError: sys.meta_path is None, Python islikely shutting down (#12)



Yes. This is actually due to lib import problem. You should not add lib. in front of the import directory, which makes it malfunction.
 
&mdash;

**Comment by cxcjcj** [comment]
Thanks for your reply. Can you tell me which files need to be modified?

**Comment by cxcjcj** [comment]
Thanks for your reply. Can you tell me which files need to be modified?

**Comment by Ze-Yang** [comment]
Actually, if you don't modify the original code, this problem won't arise. For me, at that time, I just modify _from xxx import zzz_ to _from lib.xxx import zzz_, for better debugging, which cause the above error. I am not sure whether your error is caused by the same reason with me. So please first check the import lines in the head of your execute file.

**Comment by Ze-Yang** [comment]
Actually, if you don't modify the original code, this problem won't arise. For me, at that time, I just modified _from xxx import zzz_ to _from lib.xxx import zzz_, for better debugging, which caused the above error. I am not sure whether your error is caused by the same reason with me. So please first check the import lines in the head of your execute file.

**Comment by cxcjcj** [comment]
Yes, this error only appeared on my dataset.

**Comment by JeremyLinky** [comment]
> i meet this error, too.
> Have you solved the problem?

I meet the same problem! Have you fixed it?!
